README.md

```diff
@@ -70,3 +70,9 @@ Steps to add a module to Skylib:
    ```
 
 1. Add unit tests for your module in the `tests` directory.
+
+## `skylark_library`
+
+The `skylark_library.bzl` rule allows aggregation of Skylark
+rules for use in Python-interpreter-based unit tests and documentation
```

Comment:
I don't think we want to focus on the Python-interpreter part here. The important part about `skylark_library` is that it creates a proper deps tree for a library so that _any_ test target (for example, a shell test that creates a temporary workspace) can pull in the .bzl files it depends on and ensure that all the dependencies are also available.

Suggestion: "The \`skylark_library\` rule can be used to aggregate a set of Skylark files and its dependencies for use in test targets and documentation generation."

Reply:
Done.